src/app/data-planes/components/StandardDataplanePolicies.vue

```diff
@@ -3,56 +3,62 @@
     data-testid="standard-dataplane-policies"
     class="stack"
   >
-    <KCard v-if="props.showPoliciesSection">
-      <PolicyTypeEntryList
-        id="policies"
-        :policy-type-entries="policyTypeEntries"
-        data-testid="policy-list"
-      />
+    <KCard v-if="(props.showPoliciesSection ? props.sidecarDataplanes.length === 0 : true) && props.inspectRulesForDataplane.rules.length === 0">
+      <EmptyBlock />
     </KCard>
 
-    <KCard v-if="proxyRule">
-      <h3>{{ t('data-planes.routes.item.proxy_rule') }}</h3>
-
-      <RuleEntryList
-        id="proxy-rules"
-        class="mt-2"
-        :rule-entries="[proxyRule]"
-        :show-matchers="false"
-        data-testid="proxy-rule-list"
-      />
-    </KCard>
-
-    <KCard v-if="toRuleEntries.length > 0">
-      <h3>{{ t('data-planes.routes.item.to_rules') }}</h3>
+    <template v-else>
+      <KCard v-if="props.showPoliciesSection">
+        <PolicyTypeEntryList
+          id="policies"
+          :policy-type-entries="policyTypeEntries"
+          data-testid="policy-list"
+        />
+      </KCard>
 
-      <RuleEntryList
-        id="to-rules"
-        class="mt-2"
-        :rule-entries="toRuleEntries"
-        data-testid="to-rule-list"
-      />
-    </KCard>
+      <KCard v-if="proxyRule">
+        <h3>{{ t('data-planes.routes.item.proxy_rule') }}</h3>
 
-    <KCard v-if="fromRuleInbounds.length > 0">
-      <h3 class="mb-2">
-        {{ t('data-planes.routes.item.from_rules') }}
-      </h3>
+        <RuleEntryList
+          id="proxy-rules"
+          class="mt-2"
+          :rule-entries="[proxyRule]"
+          :show-matchers="false"
+          data-testid="proxy-rule-list"
+        />
+      </KCard>
 
-      <div
-        v-for="(fromRule, index) in fromRuleInbounds"
-        :key="index"
-      >
-        <h4>{{ t('data-planes.routes.item.port', { port: fromRule.port }) }}</h4>
+      <KCard v-if="toRuleEntries.length > 0">
+        <h3>{{ t('data-planes.routes.item.to_rules') }}</h3>
 
         <RuleEntryList
-          :id="`from-rules-${index}`"
+          id="to-rules"
           class="mt-2"
-          :rule-entries="fromRule.ruleEntries"
-          :data-testid="`from-rule-list-${index}`"
+          :rule-entries="toRuleEntries"
+          data-testid="to-rule-list"
         />
-      </div>
-    </KCard>
+      </KCard>
+
+      <KCard v-if="fromRuleInbounds.length > 0">
+        <h3 class="mb-2">
+          {{ t('data-planes.routes.item.from_rules') }}
+        </h3>
+
+        <div
+          v-for="(fromRule, index) in fromRuleInbounds"
+          :key="index"
+        >
+          <h4>{{ t('data-planes.routes.item.port', { port: fromRule.port }) }}</h4>
+
+          <RuleEntryList
+            :id="`from-rules-${index}`"
```

Comment:
Is this just because `KCodeBlock` requires an `id` to be set which gets used internally? To rephrase that a little, is this id necessary for anything outside of `KCodeBlock`?

Reply:
Yes. A code block needs a unique ID to avoid label/input association clashes.

Reply:
Not really something I’m concerned about at the moment. Setting these IDs is simple enough and not something we do over and over.

Reply:
Yeah don't worry, didn't need doing here, was just checking/questioning for future.